dist-scripts/debian/gmrender-resurrect.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=gmrender-resurrect service
+After=network.target sound.target
+
+[Service]
+Environment="UPNP_DEVICE_NAME=Raspberry"
+Environment="UPNP_UUID=671ba82bb2add32f128675e011c453d0"
```

Comment:
Can you make the UUID dynamic as it is in the init.d script ? The problem with a fixed UUID is, that then all gmrender-resurrect on the same network can not be distinguished.

So if they are based on the MAC address or something (something that is predictable after reboot), that would be great.